pxr/imaging/plugin/hdRpr/rprApi.cpp

```diff
@@ -1949,13 +1949,18 @@ class HdRprApiImpl {
         float focalLength;
 
         GfVec2f apertureSize;
+        GfVec2f apertureOffset(0.0f);
         HdRprCamera::Projection projection;
         if (m_hdCamera->GetFocalLength(&focalLength) &&
             m_hdCamera->GetApertureSize(&apertureSize) &&
+            m_hdCamera->GetApertureOffset(&apertureOffset) &&
```

Comment:
Seems here apertureOffset isn't calculated for orthographic camera.

Reply:
I can confirm that apertureOffset is correctly passed in the case of the ortho camera. But it has no effect on the appearance of a scene. That's only the case for Northstar. Tahoe reacts to changes in lens shift.

Could you please try with the Full (Legacy) quality?

Reply:
Yes, I tried with Legacy quality - it works as expected, thank you for confirmation.

Reply:
https://amdrender.atlassian.net/browse/RPRNEXT-831